src/Sylius/Bundle/ApiBundle/Serializer/ProductSerializer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Serializer;
+
+use Sylius\Component\Core\Model\ProductInterface;
+use Symfony\Component\Serializer\Normalizer\ContextAwareNormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+use Webmozart\Assert\Assert;
+
+final class ProductSerializer implements ContextAwareNormalizerInterface
+{
+    /** @var NormalizerInterface */
+    private $objectNormalizer;
+
+    /** @var int */
+    private $reviewsLimit;
+
+    public function __construct(
+        NormalizerInterface $objectNormalizer,
+        int $reviewsLimit
+    ) {
+        $this->objectNormalizer = $objectNormalizer;
+        $this->reviewsLimit = $reviewsLimit;
+    }
+
+    public function normalize($object, $format = null, array $context = [])
```

Comment:
should it be a doctrine extension? like `HideArchivedShippingMethodExtension`? and IMO limit should be a API filter instead serializer

Reply:
Does the behavior of limiting resources not work by default in API Platform?

In Swagger there is a possibility to define number of product reviews:
![image](https://user-images.githubusercontent.com/6140884/112613172-19548800-8e20-11eb-8d19-2ddd32d34567.png)